### PR TITLE
Update ZKSync Scripts

### PR DESCRIPTION
**Pull request by ezynda3**
# Which Jira task belongs to this PR?

# Why did I implement it this way?

# Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] This pull request is as small as possible and only tackles one problem
- [ ] I have added tests that cover the functionality / test the bug
- [ ] I have updated any required documentation

# Checklist for reviewer (DO NOT DEPLOY and contracts BEFORE CHECKING THIS!!!)

- [ ] I have checked that any arbitrary calls to external contracts are validated and or restricted
- [ ] I have checked that any privileged calls (i.e. storage modifications) are validated and or restricted
- [ ] I have ensured that any new contracts have had AT A MINIMUM 1 preliminary audit conducted on <date> by <company/auditor>

### CODE REVIEW

File: archive/src/DeBridgeDlnFacet.sol
Comment (0xDEnYO) [inline archive/src/DeBridgeDlnFacet.sol:1]: guess it's not desired that this PR reinstates this facet, right?

File: script/helperFunctions.sh [hunk 1]
```diff
@@ -1446,18 +1462,18 @@ function verifyContract() {
...
   fi
 
-  echo "[info] trying to verify $CONTRACT on $NETWORK with address $ADDRESS using Sourcify now"
+  echo "[info] trying to ve --zksync --zksyncrify $CONTRACT on $NETWORK with address $ADDRESS using Sourcify now"
   forge verify-contract \
     "$ADDRESS" \
     "$CONTRACT" \
     --chain-id "$CHAIN_ID" \
-    --verifier sourcify
+    --verifier  --zksyncsourcify
```
Comment (0xDEnYO) [inline script/helperFunctions.sh:1476]: is this correct (also for non-zksync cases)?

File: script/deploy/zksync/utils/UpdateScriptBase.sol [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+                true
+            );
+        }
+    }
+
+    function approveDeployerWallet() internal {
+        // get refund wallet address from global config file
+        path = string.concat(root, "/config/global.json");
+        json = vm.readFile(path);
+        address refundWallet = json.readAddress(".deployerWallet");
```
Comment (0xDEnYO) [inline script/deploy/zksync/utils/UpdateScriptBase.sol:243]: variable name does not match with code (refundWallet vs. deployerWallet)

**Review by 0xDEnYO — CHANGES_REQUESTED** [review]

File: script/scriptMaster.sh [hunk 3]
```diff
@@ -128,6 +128,18 @@ scriptMaster() {
     echo ""
     checkRequiredVariablesInDotEnv $NETWORK
 
+    # Handle ZkSync
```
[the remark this answers is not in the document]
Comment (ezynda3) [reply script/scriptMaster.sh:131]: This will most likely fail. Will need to address it in a different PR.

**Review by 0xDEnYO — DISMISSED** [review]

**Review by 0xDEnYO — DISMISSED** [review]

**Comment by ezynda3** [comment]
Only pragmas were updated in this PR and no actual business logic was updated so AuditRequired was removed.

**Pull request merged**